Animal-sound/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+
+// interfata IAnimal
+interface IAnimal
+{
+    string Nume { get; }
+    string Sunet { get; }
+}
+ // clasa Animal, ce impleneteaza IAnimal
+class Animal : IAnimal
+{
+    public string Nume { get; init; }
+    public string Sunet { get; init; }
+
+// constructorul cls Animal
+    public Animal(string nume, string sunet)
+    {
+        Nume = nume;
```

Comment:
ai nevoie de un nume? "A Cat is always a Cat". Numele ei poate fi Felix, dar asta este altceva. Poti lua numele unei clase direct din instanta ei, nu?